server/build.sbt

```diff
@@ -9,6 +9,7 @@ lazy val root = (project in file("."))
     scalaVersion := "2.13.8",
     maintainer := "uat-public-contact@google.com",
     libraryDependencies ++= Seq(
+      caffeine,
```

Comment:
Suggestion:  inline doc that this provides caching utilities.

Reply:
done